images/dcos/bootstrap.sh

```diff
@@ -257,7 +268,7 @@ prepare_service ${monitor_dir} ${service_dir} controller-manager ${CONTROLLER_MA
 fdmove -c 2 1
 ${apply_uids}
 /opt/km controller-manager
-  --address=${host_ip}
+  --address=127.0.0.1
```

Comment:
may need to use address=0.0.0.0 hostname-override=127.0.0.1 so that we avoid awful hostname problems but still expose the port to the outside world (we'll want to collect metrics at some point)